coremltools/converters/mil/frontend/tensorflow/ops.py

```diff
@@ -550,9 +550,8 @@ def Cross(context, node):
 @register_tf_op
 def Einsum(context, node):
     equation = node.attr["equation"]
-    a = context[node.inputs[0]]
```

Comment:
FYI - the change in this file ended up being wrong. It was caught by an internal CI system.

The correct change would be to leave the definition of `a` and `b`, but not define `vars`. Then pass `build_einsum_mil` the value of `[a, b]` rather than `vars`.

This bug should have been caught by our public CI. I will put up a PR to fix that issue.